File: plugins/go-formatter/tests/fixtures/malformed.go
package main

import    (
"fmt"
  "strings"
)

type   Person   struct{
Name   string
Age int
}

func    (p   *Person)   GetInfo()   string   {
return   fmt.Sprintf("%s is %d years old",p.Name,p.Age)
}

func    calculate(  a,b,c   int  )int{
result:=a+b+c
return     result
}

func main(){
numbers:=[]int{1,2,3,4,5}
doubled:=make([]int,len(numbers))

for   i,num:=range   numbers{
doubled[i]=num*2
}

person:=Person{Name:"Alice",Age:30}
info:=person.GetInfo()

if   true{
fmt.Println(  "This has bad spacing"  )
x:=1+2+3
y:=x*2
fmt.Println(strings.Join([]string{"a","b","c"},","))
}
}

File: plugins/swift-formatter/tests/fixtures/malformed.swift
import    Foundation

struct   Person   {
var   name:   String
var age:Int
}

extension    Person   {
func    getInfo()   ->   String   {
return   "\(name) is \(age) years old"
}
}

func    calculate(  a:Int,b:Int,c:Int  )->Int{
let result=a+b+c
return     result
}

func main(){
let numbers=[1,2,3,4,5]
var doubled=[Int]()

for   num   in   numbers{
doubled.append(num*2)
}

let person=Person(name:"Alice",age:30)
let info=person.getInfo()

if   true{
print(  "This has bad spacing"  )
let x=1+2+3
let y=x*2
print(info)
}
}
